### ISSUE TITLE
Some transcription services are missing REST docs

### ISSUE THREAD

**Issue by lkiesow** [opencast/opencast#4074]
Amberscript and Azure transcriptions services are missing proper REST docs.
They should be fully filled out or not present any docs at all.

![Screenshot from 2022-08-13 22-18-59](https://user-images.githubusercontent.com/1008395/184509566-e90f2f04-4e6d-4706-a189-e3d5088f04a6.png)

![Screenshot from 2022-08-13 22-18-23](https://user-images.githubusercontent.com/1008395/184509567-e1b4de4d-fa90-498d-ae2d-9d6eb6e815ec.png)

**Comment by Arnei** [comment]
The docs are "correct" in the sense that the Rest Services are actually not offering any endpoints, as far as I can tell. Does that mean we can just remove them? Are they needed for anything?

Edit: Apparently they extend AbstractJobProducer, which they do need. So no removing.

**Comment by Arnei** [comment]
One easy thing we could do is removing the `@RestService` annotation from the services. Then `/transcripts/amberscript` would display this for example:

<img width="649" height="42" alt="Image" src="https://github.com/user-attachments/assets/a9eb2e3d-06b6-471f-9363-3dd7b3d66471" />

This would not remove `transcripts/amberscript` from the list under `/restdocs`.
I don't think this is a great solution. It would just make admins wonder why there are not rest docs, and maybe think it is a bug.

---

The more complicated option would be to also remove `transcripts/amberscript`  from the list under `/restdocs`. It would be a bit annoying, because the list relies on info from `/info/components`, which returns the list of REST services and user interfaces running on this host, of which the AmberscriptRestService correctly is a part because it *is* still running. So we would need to introduce some mechanism to filter the RestService out by some metric, which just does not seem worth the hastle.

---

The more I think about this, the more I think that the best solution here would be to add a note to the rest docs that says "This REST service does not offer any endpoints to external services". Making it clear that these docs are empty *intentionally*, and not removing them so admins don't wonder why they don't show up in the list.

@lkiesow thoughts? :D

**Comment by gregorydlogan** [comment]
I would agree with making the note.  If there was an easy way to remove it completely sure, but then we have to maintain that mechanism for an increasingly archaic REST doc tool.
